Sources/SwiftProtobufCore/SwiftProtobufContiguousBytes.swift

```diff
@@ -18,23 +18,34 @@ import Foundation
 /// By conforming your own types to this protocol, you will be able to pass instances of said types
 /// directly to `SwiftProtobuf.Message`'s deserialisation methods
 /// (i.e. `init(serializedBytes:)` for binary format and `init(jsonUTF8Bytes:)` for JSON).
-// TODO: extend doc to include how this will be used in serialization once that API change has been finalised.
-public protocol SwiftProtobufContiguousBytes {
+public protocol SwiftProtobufContiguousBytes: Hashable {
```

Comment:
Why was `Hashable` needed?

Reply:
Because of the `HashVisitor`. However, since we're getting rid of the generics in `Visitor` I can get rid of this conformance.